action/protocol/staking/bucket_validation.go

```diff
@@ -96,7 +96,7 @@ func validateBucketEndorsement(esm *EndorsementStateManager, bucket *VoteBucket,
 	case errors.Is(err, state.ErrStateNotExist):
 		if isEndorsed {
 			return &handleError{
-				err:           errors.New("bucket is not endorsed"),
+				err:           errors.New("bucket is not an endorse bucket"),
```

Comment:
The error here is the same as the error on line 86 above, and it is best to distinguish it

Reply:
okay